### DISCUSSION TITLE
CefSharp.BrowserSubprocess has stopped working

### DISCUSSION THREAD

**Discussion by iamkushalbh** [cefsharp/CefSharp#4620] [General]
Hi,
We are using Chromium browser by using Chromium Embedded Framework (102.0.9.0) in our application. In our application in a particular flow while clicking on a menu link we are observing the error pop-up stating "**CefSharp.BrowserSubprocess has stopped working**", same can be noticed in event viewer also. 

![MicrosoftTeams-image](https://github.com/cefsharp/CefSharp/assets/54840486/3e66e81f-9cf8-47c4-ae63-bbe5fc6360f3)

After selecting "**Close the program**" from the pop up the application it is becoming unresponsive and application page goes blank.
Message in event viewer is as follows:
**Faulting application name: CefSharp.BrowserSubprocess.exe, version: 102.0.90.0, time stamp: 0xfc820ee0
Faulting module name: libcef.dll, version: 102.0.9.0, time stamp: 0x628409ae
Exception code: 0x4000001f
Fault offset: 0x02f652e4
Faulting process id: 0x31d0
Faulting application start time: 0x01da0bde4a23f438
Faulting application path: C:\Program Files (x86)\GreenwayTR\CefSharp.BrowserSubprocess.exe
Faulting module path: C:\Program Files (x86)\GreenwayTR\libcef.dll
Report Id: 978597eb-77d1-11ee-848b-005056a3b0ca
Faulting package full name: 
Faulting package-relative application ID:** 
![MicrosoftTeams-image (1)](https://github.com/cefsharp/CefSharp/assets/54840486/564c20de-2321-4097-b427-681b48c62002)

 
As per event viewer logs the DLL crashing is "libcef.dll" (102.0.9.0). The same version was working in our application prior, without any upgrade seems to be broken.

![MicrosoftTeams-image (2)](https://github.com/cefsharp/CefSharp/assets/54840486/c8d7efa3-4986-4439-ab82-b11cadab0eef)
![Screenshot 2023-10-31 172111](https://github.com/cefsharp/CefSharp/assets/54840486/2764c49c-056f-4d8c-8574-6677b1b82d63)

Below are the system details
Operating System: Windows 10
OS Build: 17763.4974
.Net Framework: 4.5.2
Angular Version: 9
As per the below link the issue is resolved but in the higher version also still we are facing the issue. Looking for help in this.
https://github.com/cefsharp/CefSharp/issues/3670

**Comment by amaitland** [comment]
Check the log file for errors see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

Otherwise you'll need to get a crash dump see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#unmanaged-crashes-when-the-process-dies

> As per the below link the issue is resolved but in the higher version also still we are facing the issue. Looking for help in this.
> #3670

If it was working before and you haven't upgraded then it's very unlikely this is related.

More likely something has changed, e.g. windows update etc.

I'd suggest trying the current supported version M118
